src/app/app-components/courses/courses-item/courses-item.component.ts

```diff
@@ -10,10 +10,19 @@ import { Course } from 'src/app/app-models/course';
 export class CoursesItemComponent implements OnInit {
 
   @Input() courseInfo: Course;
+  @Output() deleteCourse = new EventEmitter<string>();
 
   constructor() { }
 
   ngOnInit() {
   }
 
+  onEdit() {
+    console.log('Empty handler for edit course button.');
+  }
+
+  onDelete() {
```

Comment:
If you look well, on the template you're passing the `$event` to this method. But here you're not defining it. This could cause an error when compiling with the Ahead Of Time (AOT) compilation.

The `event` here would be the native DOM event, FYI.

Reply:
I see, thanks for catching this up!